libr/asm/asm.c

```diff
@@ -246,6 +250,7 @@ R_API void r_asm_free(RAsm *a) {
 	if (!a) {
 		return;
 	}
+	// r_num_free (a->num);
```

Comment:
leak?

Reply:
RNum should be refcounted, can you add support for it and just do r_num_unref here? this way we can have shared instances between different modules and not leak or double free them

Reply:
If uncommenting that results in a double-free, still need to find out where the previous free is done

Reply:
> RNum should be refcounted ...

ok 79d65d9a4ad0d9245339c2e9ca6527f2ae6ecf9a